editor/src/clj/editor/pipeline.clj

```diff
@@ -77,7 +96,7 @@
       {:resource (workspace/make-build-resource resource)
        :build-fn build-protobuf
        :user-data {:pb-class pb-class
-                   :pb-msg (reduce dissoc pb-msg resource-keys)
+                   :pb-msg pb-msg
```

Comment:
Yes, keeping them is safe because we store `:resource-keys` in `:user-data` which are derived from the previously dissoc-ed resources in `pb-msg`. I removed `dissoc` here because it does not affect much (hash would be different, but not in a meaningful way), but a simple `dissoc` would no longer work here (we now use paths like `[:samplers 0 :texture]`)